app/models/mars_manifest.rb

```diff
@@ -52,7 +52,9 @@ def validate_manifest
     #   nil if an error occurs.
     def raw_data
       # TODO: Stop bypassing SSL check.
-      @raw_data ||=  open(url, { ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE }).read
+
+      # sub out vertical tabs, they are expected in mars exports
+      @raw_data ||=  open(url, { ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE }).read.gsub(/\t/, "\n")
```

Comment:
Won't newline characters be recognized as new records?

Also, isn't `\t` just a regular tab (not vertical tab)?

I think logically, what we want to do is replace vertical tabs with just a space. If we're flattening data down to a CSV where newline characters mean new records, then we can't allow newline characters within the record. Right?

Reply:
the newlines should get handled properly within a description if they're in double-quotes

Reply:
\t is any tab character, so it covers vt but you may be right that a regular tab shouldn't be turned into a newline

Reply:
But doesn't that mean that an unquoted vertical tab (or as written, an unquoted any tab) will cause chaos?